doc/doc.go

```diff
@@ -0,0 +1,71 @@
+package doc
+
+import (
+	"html/template"
+	"io"
+	"net/http"
+
+	"github.com/labstack/echo"
+)
+
+const (
+	defaultDocURL = "doc"
+)
+
+type params struct {
+	docURL          string
+	swaggerFilePath string
+}
+
+type Template struct {
+	templates *template.Template
+}
+
+func (t *Template) Render(w io.Writer, name string, data interface{}, c echo.Context) error {
+	return t.templates.ExecuteTemplate(w, name, data)
+}
+
+func newHTMLRenderer() (*Template, error) {
+	t, err := template.New(defaultDocURL).Parse(htmlTemplate)
```

Comment:
mb this will be better? https://golang.org/pkg/html/template/#example_Template_parsefiles

Reply:
and why I cannot configure this?

Reply:
nope, because you will need to add files with `parsefiles`, and this is an external lib, you will not be able to do the forwarding to the docker this files, to run app:
like this: 
`COPY --from=builder /go/bin/yaga/html.files /yaga/html.files`
And now everything will work without changing the docker

Reply:
oh, no man... I not mean this..
I talk about configuration